File: module_java/src/main/java/com/example/module_java/MainActivity.java
package com.example.module_java;


import android.os.Bundle;

import com.alibaba.android.arouter.facade.annotation.Route;
import com.example.common.content.PagePath;
import com.example.common.ui.base.ARouterActivity;

@Route(path = PagePath.ModuleJavaPage.MAIN_PAGE)
public class MainActivity extends ARouterActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

    }
}

File: module_kotlin/src/main/java/com/example/module_kotlin/MainActivity.kt
package com.example.module_kotlin

import android.os.Bundle
import android.widget.Button
import com.alibaba.android.arouter.facade.annotation.Route
import com.alibaba.android.arouter.launcher.ARouter
import com.example.common.content.PagePath
import com.example.common.ui.base.ARouterActivity


@Route(path = PagePath.ModuleKotlinPage.MAIN_PAGE)
class MainActivity : ARouterActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_kotlin_main)

        findViewById<Button>(R.id.bt_click_main).setOnClickListener {
            ARouter.getInstance().build(PagePath.ModuleKotlinPage.FUNCTION_PAGE).navigation()
        }
    }
}